apps/et/et-cos/perftest-image-policy.yaml

```diff
@@ -2,14 +2,6 @@ apiVersion: image.toolkit.fluxcd.io/v1beta1
 kind: ImagePolicy
```

Comment:
Please leave this configuration in and change the image policy to point to master instead.